docs/api/programme.md

```diff
@@ -0,0 +1,251 @@
+# Programme
+
+A programme represents a type of graduation such as LEIC or MEIC. 
+A programme is composed by a set of _offers_. 
+
+## Properties
+   * Id
+      - MANDATORY
+      - type: number
+      - Identifies uniquely a Programme
+   * Name   
+      -  OPTIONAL (it may not appear in this phase)
+      -  type: text
+      -  e.g. "Licenciatura em Engenharia Informática e de Computadores"
+    
+   * Acronym: programme's acronym
+      - mandatory
+      - type: text
+      - e.g. "LEIC"
+
+   * Term Size: Duration in terms
+      - mandatory
+      - type: number
+      - e.g. 6 terms for LEIC, 4 terms for MEIC
+
+
+## Link Relations
+   * A programme representation
+      - must include a link to its context, using the self link relation.
+      - may include a link to the collection it belongs, using the collection link relation.
+      - must include a link to its _offers_.
+
+## Actions
+    * edit: edit's a programme
+    * add Offer: add's a curricular unit offer to the programme
+
+## Example representation
+```json
+{
+    "class": [ "programme" ],
+    "properties": {
+        "id": 1,
+        "name": "Licenciatura em Engenharia Informática e de Computadores", //It may not appear
+        "acronym": "LEIC",
+        "termSize": 6
+    },
+    "entities": [
+        {
+            "class": [ "offer" ],
+            "title": "LS Offer",
+            "rel": [ "/rel/programmeOffer" ],
+            "properties": {
+                "CourseId": 2,
+                "TermNumber": 3
+            },
+            "links" : [
+                { "rel": [ "self" ], "href": "/v0/programmes/LEIC/offers/1"}
+            ]
+        },
+        {
+            "class": [ "offer" ],
+            "title": "AED Offer",
+            "rel": [ "/rel/programmeOffer" ],
+            "properties": {
+                "CourseId": 5,
+                "TermNumber": 3
+            },
+            "links" : [
+                { "rel": [ "self" ], "href": "/v0/programmes/LEIC/offers/2"}
+            ]
+        },
+        {
+            "class": [ "offer" ],
+            "title": "POO Offer",
+            "rel": [ "/rel/programmeOffer" ],
+            "properties": {
+                "CourseId": 4,
```

Comment:
As an embed, I think the Term Number shouldn't be here.
Also, properties' object should have the offer's ID.